README.md

```diff
@@ -56,8 +56,25 @@ $ make test
 
 In order to run the full suite of Acceptance tests, run `make testacc`.
 
-*Note:* Acceptance tests create real resources, and often cost money to run.
+*Note:* 
+- Acceptance tests create real resources, and often cost money to run.
+- If ran locally `docker-compose` needs to be in the `$PATH`
 
 ```sh
 $ make testacc
 ```
+
+In order to manually run some Acceptance test locally, run the following commands:
+```sh
+# spins up a local docker postgres container
+make testacc_setup 
+
+# Load the needed environment variables for the tests
+source tests/env.sh
```

Comment:
Could it be directly done in the ``testacc_setup`` script?

Reply:
It is already read in the `setup` but it's not exported into the calling shell because the make target is run in in a subshell. You will have to read those `env` once in the shell you want to iterate over the tests ...